packages/office-ui-fabric-react/src/components/SearchBox/examples/SearchBox.Nonclearable.Example.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { SearchBox } from 'office-ui-fabric-react/lib/SearchBox';
+import './SearchBox.Examples.scss';
```

Comment:
I'm about to make another change simplifying the SearchBox examples and getting rid of the SCSS. To go along with that, please update this example as follows:

```tsx
import * as React from 'react';
import { SearchBox } from 'office-ui-fabric-react/lib/SearchBox';

// tslint:disable:jsx-no-lambda

export const SearchBoxNonclearableExample: React.FunctionComponent = () => {
  return (
    <SearchBox
      clearable={false}
      placeholder="Search"
      onSearch={newValue => console.log('value is ' + newValue)}
      onFocus={() => console.log('onFocus called')}
      onBlur={() => console.log('onBlur called')}
      onChange={() => console.log('onChange called')}
    />
  );
}
```